routing/cross_mwm_ids.hpp

```diff
@@ -0,0 +1,72 @@
+#pragma once
+
+#include "routing_common/transit_types.hpp"
+
+#include "base/newtype.hpp"
+#include "base/visitor.hpp"
+
+#include <cstdint>
+#include <functional>
+
+namespace routing
+{
+namespace connector
+{
+// Identifier to find a border edge in neighbouring mwm while cross mwm transition
+// for pedestrian, bicycle and car routing.
+NEWTYPE(uint64_t, OsmId);
```

Comment:
Please, consider to re-use OsmId from generator.

Reply:
Done